Docs update: View (#1924)
aws-amplify/amplify-ui: docs/src/pages/[platform]/components/view/index.page.mdx

---
title: View

> description: View is the container component on top of which all other Amplify UI components live.

Reviewer: ```suggestion
description: View is the container component within which all other Amplify UI components live.
```

Reviewer: Maybe we can include that `View` renders a `div` by default here. Something like:
```
description: View is the container component within which all other Amplify UI components live. It renders a <div> as the container by default, but can be set to render any HTML element.
```
